spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Sequential.scala

```diff
@@ -369,6 +397,26 @@ class Sequential extends MklDnnContainer with MklInt8Convertible {
     }
   }
 
+  override def calcScales(input: Activity): Unit = {
+    var i = 0
+    var lastOutput = input
+    while (i < this.modules.length - 1) {
+      this.modules(i) match {
+        case mkldnnModule: MklInt8Convertible => mkldnnModule.calcScales(lastOutput)
+        case _ =>
+      }
+
+      val curOutput = this.modules(i).output
+      require(mklDnnModules(i).outputFormats().length == mklDnnModules(i + 1).inputFormats().length)
+      lastOutput = reorderManager.infer(
+        mklDnnModules(i).outputFormats(),
+        mklDnnModules(i + 1).inputFormats(),
+        curOutput
+      )
+
+      i += 1
+    }
+  }
```

Comment:
Think about the case:  The Sequential module has only 1 sub-module.
The first time we are trying to enter the while loop, condition (0 < 1 - 1) is false.

Reply:
Yes. You are right. I step into the same river again.